tools/deprecated-config-checker/deleted-config.yaml

```diff
@@ -0,0 +1,33 @@
+# This file should contain a list fo deleted config options.
+#
+# The syntax for this file is the same as the syntax for deprecated-config.yaml.
+
+ingester:
+  max_transfer_retries: "Enable the ingester WAL and rely on new ingesters to replay the WAL."
+
+querier:
+  engine:
+    timeout: "Configure global or per-tenant query_timeout limit."
+
+query_range:
+  split_queries_by_interval: "Configure global or per-tenant split_queries_by_interval limit"
+  forward_headers_list: "This setting is removed without an alternative"
+
+frontend_worker:
+  parallelism: "Configure querier.max_concurrent to limit the max concurrent requests processed by the queriers."
+  match_max_concurrent: "Configure querier.max_concurrent to limit the max concurrent requests processed by the queriers."
+
+common:
+  storage:
+    s3: &s3_deletes
```

Comment:
`s3_deletes`? is this part of the syntax.

Reply:
I'm using YAML [aliases and anchors][1] so I don't repeat the deprecated fields for s3. 

[1]: https://www.educative.io/blog/advanced-yaml-syntax-cheatsheet#anchors

Reply:
aha, nice